Fix lint warnings and specify compiler version

DemiMarie opened the pull request:
Truffle needs a specific compiler version to be specified.  Also, some
lint warnings could be fixed.

DemiMarie committed:
Fix lint warnings and specify compiler version

Truffle needs a specific compiler version to be specified.  Also, some
lint warnings could be fixed.

drewstone commented on contracts/EdgewareERC20.sol:
```diff
@@ -5,20 +5,21 @@ import "openzeppelin-solidity/contracts/token/ERC20/ERC20Mintable.sol";
 
     /**
     * @dev Burns a specific amount of tokens.
+     * @param account The address to burn tokens from.
```
Spacing is off by 1.

drewstone commented on contracts/ValidatorSet.sol:
```diff
@@ -32,27 +32,28 @@ contract ValidatorSet {
         }
         address recAddr = ecrecover(hash, v[i], r[i], s[i]);
-        require(recAddr == addresses[signers[i]]);
+            require(recAddr == addresses[signers[i]], "Invalid recipient address");
```
Spacing is off.

DemiMarie replied:
Fixed.

drewstone reviewed (changes requested):
Have you run all tests and ensured they pass?

DemiMarie commented:
@drewstone I tried to run the tests, but failed.  What is the best way to run them?

DemiMarie commented:
@drewstone Running `truffle test` fails on master and on this branch, with obscure errors in both cases.

DemiMarie commented:
That said, I am not sure what version of Truffle to use.

DemiMarie committed:
Fix test suite

Truffle 5.0 is not backwards-compatible with earlier versions, requiring
many changes.

DemiMarie commented:
@drewstone The tests now pass on my system (Truffle 5.0.0).

DemiMarie committed:
Commit package-lock.json

drewstone commented:
Look good! Thanks @DemiMarie !
